recipes/lscsoft-glue/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "lscsoft-glue" %}
+{% set version = "1.58.2" %}
+{% set sha256 = "7002676d2989f959bfae53cce622eb528acc82cc9d43e76a2c850e1577e1aaf4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python {{ python }}
```

Comment:
can you remove the jinja bit here,  It is not needed since these are added implicitly by conda-render